security/subsystem/src/main/resources/schema/jboss-as-security_2_0.xsd

```diff
@@ -44,6 +44,7 @@
          <xs:element name="security-management" type="securityManagementType" minOccurs="0" maxOccurs="1"/>
          <xs:element name="security-domains" type="securityDomainsType" minOccurs="0" maxOccurs="1"/>
          <xs:element name="vault" type="vaultType" minOccurs="0" maxOccurs="1"/>
+         <xs:element name="elytron-integration" type="elytronIntegrationType" minOccurs="0" maxOccurs="1"/>
```

Comment:
Makes sense, I think. If nothing changes for WF 10 (i.e. the schema remains at version 1.x) we can always change this commit again and downgrade the schema to 2.0 for WF 11.